senior_teacher.rb

```diff
@@ -1,53 +1,19 @@
-class SeniorTeacher
-  attr_reader :age, :salary, :phase, :performance_rating, :target_raise
-  attr_accessor :name
+require_relative 'teacher'
 
-  def initialize(options={})
-    @phase = 3
-    @age = options.fetch(:age, 0)
-    @name = options.fetch(:name, "")
-    @target_raise = 1000
-  end
-
-  def offer_high_five
-    "High five!"
-  end
-
-  def set_phase(num)
-    @phase = num
-    "Cool, I've always wanted to teach phase #{num}!"
-  end
+class SeniorTeacher < Teacher
+  attr_reader :performance_rating
 
-  def teach_stuff
-    response = ""
-    response += "Listen, class, this is how everything works, fo SHO! "
-    response += "*drops flat-out insane knowledge bomb* "
-    response += "... You're welcome. *saunters away*"
-    response
-  end
-
-  def salary=(new_salary)
-    puts "This better be good!"
-    @salary = new_salary
-  end
+  # here I was hesitating between using interpolation with #{, fo SHO!} 
+  # and #{*saunters away*} and then set this in teacher method but couldn't decide what is better practice
```

Comment:
Honestly, this is more of a personal preference thing.  I think I would do the interpolation version but it is not very important.